crates/flux-config/src/lib.rs

```diff
@@ -47,6 +47,10 @@ pub fn is_cache_enabled() -> bool {
     CONFIG.cache
 }
 
+pub fn is_checked_file(file: &str) -> bool {
```

Comment:
If we parse it as a list of files with can make this take a `Path`

```suggestion
pub fn is_checked_file(file: &Path) -> bool {
```

Reply:
and we should canonicalize paths (if possible) before comparing them.